src/wallet/rpcdump.cpp

```diff
@@ -1015,6 +1032,12 @@ UniValue importmulti(const JSONRPCRequest& mainRequest)
     LOCK2(cs_main, pwalletMain->cs_wallet);
     EnsureWalletIsUnlocked();
 
+    // Verify all timestamps are present before importing any keys.
+    const int64_t now = chainActive.Tip() ? chainActive.Tip()->GetMedianTimePast() : 0;
```

Comment:
I don't know if 0 is a good default for `now`, but it doesn't seem harmful either. I wonder if the function shouldn't simply throw a `runtime_exception` if `!chainActive.Tip()` saying to sync up first..